daemon/seccomp_disabled.go

```diff
@@ -0,0 +1,12 @@
+// +build !seccomp,!windows
```

Comment:
This was causing me an error when trying to build the packages. with @calavera 's help, he helped me get past the error. We changed the line to the following.

```
// +build !seccomp
```

We should double check that is what we want, and if so, make the change.